### PR TITLE
Add: Sandbox dataset description + minor changes

### PR DESCRIPTION
**Pull request by nikolaj-encord**

### CODE REVIEW

File: docs/docs/cli/download-sandbox-data.md [hunk 1]
```diff
@@ -15,3 +15,91 @@ The script will ask you to choose a project, navigate the options with <kbd>↑<
 
 When the download process is complete, you visualise the results by following the printed instructions.
 For more details, see the [Open Encord Active](/cli/open-encord-active) page.
+
+
+If you have a dataset you'd like us to include in the sandbox please submit an issue on Github or contact us directly. We'd love to help!
```
Comment (frederik-encord) [inline docs/docs/cli/download-sandbox-data.md:20]:
```suggestion
If you have a dataset you'd like us to include in the sandbox please submit an [issue](https://github.com/encord-team/encord-active/issues) on Github or contact us directly at [active@encord.com](mailto:active@encord.com). 
We'd love to help!
```

File: docs/docs/cli/download-sandbox-data.md [hunk 2]
```diff
@@ -15,3 +15,91 @@ The script will ask you to choose a project, navigate the options with <kbd>↑<
...
+![BDD dataset](../images/BDD.png)
+
+## COCO Validation 2017 Dataset
+
+ **About the dataset**
+
+- **Research Paper:** [Microsoft COCO: Common Objects in Context](https://arxiv.org/abs/1405.0312)
+- **Author:** [Tsung-Yi Lin](https://arxiv.org/search/cs?searchtype=author&query=Lin%2C+T) , [Michael Maire](https://arxiv.org/search/cs?searchtype=author&query=Maire%2C+M), [Serge Belongie](https://arxiv.org/search/cs?searchtype=author&query=Belongie%2C+S), [Lubomir Bourdev](https://arxiv.org/search/cs?searchtype=author&query=Bourdev%2C+L) , [Ross Girshick](https://arxiv.org/search/cs?searchtype=author&query=Girshick%2C+R), [James Hays](https://arxiv.org/search/cs?searchtype=author&query=Hays%2C+J), [Pietro Perona](https://arxiv.org/search/cs?searchtype=author&query=Perona%2C+P), [Deva Ramanan](https://arxiv.org/search/cs?searchtype=author&query=Ramanan%2C+D), [C. Lawrence Zitnic](https://arxiv.org/search/cs?searchtype=author&query=Zitnick%2C+C+L), [Piotr Dollár](https://arxiv.org/search/cs?searchtype=author&query=Doll%C3%A1r%2C+P)
+- **Dataset Size:**  5000 images, 4784 annotations
+- **Categories:**  7 plane categories
```
Comment (frederik-encord) [inline docs/docs/cli/download-sandbox-data.md:44]: 81 classes no? Dont know what the 7 planes would be? Is that leftover from `rareplanes`?

File: docs/docs/cli/download-sandbox-data.md [hunk 3]
```diff
@@ -15,3 +15,91 @@ The script will ask you to choose a project, navigate the options with <kbd>↑<
...
+
+Sample pictures:
+![COCO dataset](../images/COCO.png)
+
+## Covid 19 Segmentation Dataset
+
+**About the dataset**
+
+- **Research Paper: None**
+- **Author:**
```
Comment (frederik-encord) [inline docs/docs/cli/download-sandbox-data.md:58]: Perhaps add unknown or something?

File: docs/docs/cli/download-sandbox-data.md [hunk 4]
```diff
@@ -15,3 +15,91 @@ The script will ask you to choose a project, navigate the options with <kbd>↑<
...
+- **Dataset Size:** 2710 images & 6812 annotations
+- **Categories:**  10 categoties
+- **License: CC BY 4.0**
+- **Release: XYZ**
+- **Read more:** [Webpage](https://www.cosmiqworks.org/rareplanes/) & [GitHub](https://github.com/GeneralBlockchain/covid-19-chest-xray-segmentations-dataset)
+
```
Comment (frederik-encord) [inline docs/docs/cli/download-sandbox-data.md:64]: No examples?

File: docs/docs/cli/download-sandbox-data.md [hunk 5]
```diff
@@ -15,3 +15,91 @@ The script will ask you to choose a project, navigate the options with <kbd>↑<
...
+## Limuc Ulcerative Colitis Classification
+
+- **Research Paper:** Improving the Computer-Aided Estimation of Ulcerative Colitis Severity According to Mayo Endoscopic Score by Using Regression-Based Deep Learning
+- **Authors:** Gorkem Polat, MSc, Haluk Tarik Kani, MD, Ilkay Ergenc, MD, Yesim Ozen Alahdab, MD, Alptekin Temizel, PhD, Ozlen Atug, MD
+- **Dataset Size:** 11276 images
+- **Categories:** Medical (Endoscopy/Colonoscopy)
+- **License:** Creative Commons Attribution 4.0 International
+- **Release: 14/03/2022**
+- **Read more:** [Webpage](https://www.notion.so/Jan-9-2023-c23ea70fe8b64b889dc8b8ad4f77ad18) & [GitHub](https://github.com/GorkemP/labeled-images-for-ulcerative-colitis)
```
Comment (frederik-encord) [inline docs/docs/cli/download-sandbox-data.md:105]: No examples? I suppose it is on purpose on this one?

File: docs/docs/cli/initialising-project-from-image-directories.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 sidebar_position: 3
 ---
 
-# Initialise from Image Directory
+# Initialising from Image Directory
```
Comment (frederik-encord) [inline docs/docs/cli/initialising-project-from-image-directories.md:5]:
```suggestion
# Initializing from Image Directory
```

**Review by frederik-encord — CHANGES_REQUESTED** [review]

**Review by nikolaj-encord — COMMENTED** [review]
Accepted requested changes. I have pulled it out as a draft so I can add the information on the dataset that is missing.

**Review by nikolaj-encord — COMMENTED** [review]
All comments approved from my side Fred.

**Review by frederik-encord — APPROVED** [review]

**Pull request merged**